docs/static/filebeat_modules/mysql/error/pipeline.conf

```diff
@@ -0,0 +1,34 @@
+input {
```

Comment:
It seems like I get a more granular parsing of the fields when I use Filebeat Modules instead of the LS config. 

When I use FBM, I get documents that look like this (note the the error message is broken into several fields):

`````
{
  "_index": "filebeat-2017.03.21",
  "_type": "log",
  "_id": "AVryLpOCu-hv5J-8awz_",
  "_score": null,
  "_source": {
    "@timestamp": "2016-12-12T11:40:39.540Z",
    "offset": 25969,
    "beat": {
      "hostname": "Rhodas-MacBook-Pro.local",
      "name": "Rhodas-MacBook-Pro.local",
      "version": "5.3.0"
    },
    "input_type": "log",
    "mysql": {
      "error": {
        "thread_id": "0",
        "level": "Note",
        "message": "/usr/local/Cellar/mysql/5.7.10/bin/mysqld: ready for connections.",
        "timestamp": "2016-12-12T11:40:39.540498Z"
      }
    },
    "source": "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/filebeat-5.3.0-darwin-x86_64/log_examples/mysql_error/mysql-darwin-brew-5.7.10.log",
    "fileset": {
      "module": "mysql",
      "name": "error"
    },
    "type": "log"
  },
  "fields": {
    "@timestamp": [
      1481542839540
    ]
  },
  "sort": [
    1481542839540
  ]
}
`````

When I use this LS config, I get: 
`````
{
  "_index": "filebeat-2017.03.21",
  "_type": "log",
  "_id": "AVryMjBju-hv5J-8aw0s",
  "_score": null,
  "_source": {
    "@timestamp": "2017-03-21T18:47:34.593Z",
    "offset": 3991,
    "@version": "1",
    "beat": {
      "hostname": "Rhodas-MacBook-Pro.local",
      "name": "Rhodas-MacBook-Pro.local",
      "version": "5.3.0"
    },
    "input_type": "log",
    "host": "Rhodas-MacBook-Pro.local",
    "mysql": {
      "error": {
        "message": "2016-12-09T12:08:33.784722Z 0 [Note] /usr/local/Cellar/mysql/5.7.10/bin/mysqld: ready for connections."
      }
    },
    "source": "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/filebeat-5.3.0-darwin-x86_64/log_examples/mysql_error/mysql-darwin-brew-5.7.10.log",
    "type": "log",
    "tags": [
      "beats_input_codec_plain_applied"
    ]
  },
  "fields": {
    "@timestamp": [
      1490122054593
    ]
  },
  "sort": [
    1490122054593
  ]
}
`````